tests/bson-serialize-readconcern.phpt

```diff
@@ -0,0 +1,21 @@
+--TEST--
+MongoDB\Driver\ReadConcern serialization
+--FILE--
+<?php
+$tests = [
+	new \MongoDB\Driver\ReadConcern(),
+	new \MongoDB\Driver\ReadConcern( \MongoDB\Driver\ReadConcern::LOCAL ),
+	new \MongoDB\Driver\ReadConcern( \MongoDB\Driver\ReadConcern::MAJORITY ),
+	new \MongoDB\Driver\ReadConcern( \MongoDB\Driver\ReadConcern::LINEARIZABLE ),
+];
+
+foreach ( $tests as $test )
+{
+	var_dump( MongoDB\BSON\toJson( MongoDB\BSON\fromPHP( [ 'rc' => $test ] ) ) );
```

Comment:
I thought the conclusion was that we didn't care about the root, so the test leaves it as "undefined" :-)